samples/client/petstore/kotlin-retrofit2-kotlinx_serialization/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt

```diff
@@ -23,6 +23,11 @@ class ApiClient(
     private var baseUrl: String = defaultBasePath,
     private val okHttpClientBuilder: OkHttpClient.Builder? = null,
     private val callFactory : Call.Factory? = null,
+    private val converterFactories: List<Converter.Factory> = listOf(
+        ScalarsConverterFactory.create(),
+        kotlinxSerializationJson.asConverterFactory("application/json".toMediaType()),
+    ),
+    @Deprecated("this property is deprecated. use converterFactories.")
```

Comment:
@dylan-kwon Hi, first of all thanks for creating this PR.
This PR contains a breaking change.
If anyone continues to use `converterFactory`, it will stop working.
Can you please target the branch `7.0.x` instead of the master branch?
Thanks

Reply:
Changed to branch `7.0.x`.